build.gradle

```diff
@@ -1,16 +1,23 @@
+apply plugin: 'rg-multi-project'
+
 buildscript {
     repositories {
-        jcenter()
+        //jcenter()
```

Comment:
I think you should roll back this changes because those are researchgate specific

Reply:
got it, that makes sense 👍